src/navigator/TabNavigator.tsx

```diff
@@ -58,7 +58,7 @@ export default function TabNavigator({ route }: Props) {
           freezeOnBlur: false,
           lazy: false,
           tabBarLabel: t('bottomTabsNavigator.home.tabName') as string,
-          tabBarIcon: ValoraV,
+          tabBarIcon: Logo,
```

Comment:
The drawer nav uses Home icon (it's only usage) which is essentially the same as Logo in the valora branding. It is a slightly different icon in the celo branding. I don't think we need it here and remove the Home icon when we remove the drawer, but I can update this to use Home if there's a strong reason to keep it separate.